packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/domain_data_sources/count_data_sources/count_data_source.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2021, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import '../../../../base/models/count.dart';
+
+/// A contract class containing all methods,
+/// that  should be implemented by real data source
+abstract class CountDataSource{
+  Future<Count> getCurrent();
```

Comment:
@RomanovaPrime, please add comments to each method.

Reply:
Added.